.github/workflows/packages.yaml

```diff
@@ -33,10 +40,24 @@ jobs:
           - centos7-aarch64
           - centos7-x86_64
           - centos8-ppc64le
+        ispr:
+          - ${{github.event_name == 'pull_request'}}
+        exclude:
+          - ispr: true
+            package: ubuntu18.04-arm64
+          - ispr: true
+            package: ubuntu18.04-ppc64le
+          - ispr: true
+            package: centos7-aarch64
+          - ispr: true
+            package: centos8-ppc64le
       fail-fast: false
     steps:
       - uses: actions/checkout@v4
         name: Check out code
+        with:
+          fetch-tags: true
+          fetch-depth: 1024
```

Comment:
I'm not quite happy with this solution, but it's needed to allow `git describe --tags` to work.